index-de.html

```diff
@@ -0,0 +1,273 @@
+    </head>
+    <body>
+        <!--[if lt IE 8]>
+            <p class="browserupgrade">You are using an <strong>outdated</strong> browser. Please <a href="http://browsehappy.com/">upgrade your browser</a> to improve your experience.</p>
+        <![endif]-->
+
+        <h1>Guild Wars 2 - Gold pro Stunde</h1>
+
+        <div class="language">
+            <a href="index.html"><img src="img/us.svg" alt="English" title="English" width="20px"></a>
+            <a href="index-fr.html"><img src="img/fr.svg" alt="Français" title="Français" width="20px"></a>
+        </div>
+        </br>
+
+        <div id="intro">
+            <h2>Willkommen zu Guild Wars 2 - Gold pro Stunde</h2>
+            <p>Diese Applikation verfolgt (<em>beinahe</em>) alles, was auf deinem Account vor sich geht und zeigt an, wie viel Gold du bekommen würdest, wenn du alles verkaufen würdest, was du erhalten hast, und wie viel Gold du verloren hast durch die Items, die von deinem Account entfernt wurde (verkauft, zerstört, verbraucht, usw.).</p>
+            <p>Ich habe gesagt <em>beinahe</em>, weil ich bisher nicht Items und Gold beobachten kann, das im Handelsposten darauf wartet aufgenommen zu werden. Sobald ArenaNet eine neue API hinzugefügt, um diese Informationen zu erhalten, werde ich die Applikation updaten.</p>
+            <p>Sei dir bewusst, dass in einigen Fällen es dazu kommen kann, dass es 2 oder 3 Refreshes benötigt bis erhaltene oder verlorene Items auftauchen. Die API gibt keine Live Daten zurück, jedes mal wenn sie aufgerufen wird. Sei also bitte geduldig :)</p>
+            <hr>
+            <h2>Um zu beginnen, gebe hier einen API Schlüssel ein:</h2>
+            <input type="text" id="apiKey" maxwidth="100" value="">
+            <button id="continueIntro">Auf gehts!</button>
+            <br>
+            <label class="small" for="saveAPIKey"><input type="checkbox" id="saveAPIKey"> Speichere meinen API Schlüssel lokal, damit ich ihn nächstes Mal nicht eingeben muss</label>
+
+            <p>Du besitzt keinen? Gehe auf <a href="https://www.guildwars2.com/" target="_blank">guildwars2.com</a> und logge dich in deinen Account ein. Dann erstelle im <strong>Applikationen</strong> Tab einen neuen API Schlüssel. Achte darauf, dass du die Berechtigungen für <em>wallet, tradingpost, account, inventories</em> und <em>characters</em> gesetzt hast.</p>
+            <div class="borderTop"></div>
+            <div class="borderRight"></div>
+            <div class="borderBottom"></div>
+            <div class="borderLeft"></div>
+        </div>
+        
+        <div id="menu" style="display:none;">
+            <span id="about"><img src="../Guild-Wars-2-Gold-per-hourGN/img/about.png" alt="about" title="About">About</span>
+            <span class="separator">|</span>
+            <span id="settings"><img src="../Guild-Wars-2-Gold-per-hourGN/img/settings.png" alt="settings" title="Settings">Einstellungen</span>
```

Comment:
The "about" and "setting" image paths should be relative. src="img/about.png" and src="settings.png".